bin/v3c

```diff
@@ -1,9 +0,0 @@
-#!/bin/bash
```

Comment:
You've accidentally checked in your version of `bin/v3c`.

Reply:
The incantation to fix this is `git update-index --assume-unchanged bin/v3c`.

Reply:
Why not just add it to `.gitignore`? Unlikely that the script will have any changes, it's pretty simple.

Reply:
Hmm, I just tried that and it doesn't want to ignore it. I guess `.gitignore` doesn't really apply to checked-in files.

Reply:
Hm yeah. I just tried it too and it didn't work. Oh well.